js/content/store.js

```diff
@@ -3277,7 +3461,7 @@ let UserNotes = (function(){
                 if (e.key === "Enter") {
                     $J(".es_note_modal_submit").click();
                 } else if (e.key === "Escape") {
-                    Modal.Dismiss();
+                    M   odal.Dismiss();
```

Comment:
What's this?

Reply:
No idea...

Reply:
How about using ESLint to check for errors like this? At the same time, it will help making everything more consistent. Do you want me to suggest a ESLint config?

Reply:
I wanted to add some kind of linter/code checker since like beginning, but didn't know which one. We can try ESLint, if you have good config create an issue for it and add it there, please.